gitlab/visualpr.sh

```diff
@@ -124,6 +124,7 @@ do
         urlpath=$(sed "s/$prefix//g"<<<$file)
         # Small risk of collision
         storepath=$(sed "s/\//_s_/g"<<<$urlpath)
+        firefox -screenshot /dev/zero http://localhost/$url/$urlpath
```

Comment:
This should be `/dev/null` right? And perhaps add a comment why you are doing this.

Reply:
Although both will do what I need, I think the `/dev/null` is indeed better together with the comment. Will fix.